common/workunit/workunit.cpp

```diff
@@ -1042,6 +1043,7 @@ class CDaliWorkUnit : public CPersistedWorkUnit
         VStringBuffer path("/GraphProgress/%s", queryWuid());
         Owned<IRemoteConnection> conn = querySDS().connect(path, myProcessSession(), RTM_LOCK_WRITE|RTM_CREATE_QUERY, SDS_LOCK_TIMEOUT);
         IPTree *progress = ensurePTree(conn->queryRoot(), graphName);
+        // Actually, maybe it does work, but by luck?
```

Comment:
So it works because this state flag (_state) and it's structure in /GraphProgress/<wuid>/<graphname>/_state was left outside of the scope of the general stats collection+committment.

Apart from being at odds with the new stats mechanism, is there any harm?
A separate JIRA/PR should version this info and transition it over to use IStat*